jpa/eclipselink/pom.xml

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+	<parent>
+		<groupId>org.springframework.data.examples</groupId>
+		<artifactId>spring-data-jpa-examples</artifactId>
+		<version>1.0.0.BUILD-SNAPSHOT</version>
+	</parent>
+
+	<repositories>
```

Comment:
Hi @olivergierke , I have a follow up question! 

I debated using this plugin originally. The static weaving plugin listed in the EclipseLink documentation requires a persistence-unit defined in a persistence.xml. Without specifying the entity classes or a jar in the persistence-unit, it didn't actually do any weaving on the class files. The plugin in the current version of the PR does class scanning which seems more ideal, but in hindsight it's probably not ideal for an example like this because it's non standard and may stop working at some point if that non maven central plugin disappears.

I have the example working with all the other review comments incorporated so it is much more correct in terms of Spring Boot usage. I have it working using both this plugin and the standard plugin. To get the standard plugin working, I had to create a persistence.xml file and define a persistence-unit and list the entity class. The JIRA issue did state "Ideally this would include static weaving." So I can either resubmit with the standard plugin and the persistence.xml, or just remove the static weaving portion. Which would you be better?